libkpod/remove.go

```diff
@@ -0,0 +1,62 @@
+package libkpod
+
+import (
+	"fmt"
+
+	"github.com/kubernetes-incubator/cri-o/oci"
+	"github.com/sirupsen/logrus"
+)
+
+// RemoveContainers removes the given containers. If the container is running, the container
+// should be force removed.
+func (c *ContainerServer) RemoveContainers(containers []string, force bool) string {
```

Comment:
Why make this endpoint remove a list of containers? Docker's remove container API endpoint targets a single container, and then it implements handling for multiple containers in the CLI. I think that's the right model here - keeps the CLI simple.